lib/dynamo/dynamo/jobs.py

```diff
@@ -20,31 +20,46 @@ def _get_job_count_for_month(user) -> int:
     return job_count_for_month
 
 
-def get_quota_status(user) -> Tuple[int, int, int]:
+def get_quota_status(user) -> Union[tuple[int, int, int], tuple[None, None, None]]:
     max_jobs = get_max_jobs_per_month(user)
-    previous_jobs = _get_job_count_for_month(user)
-    remaining_jobs = max(max_jobs - previous_jobs, 0)
+
+    if max_jobs is not None:
+        previous_jobs = _get_job_count_for_month(user)
+        remaining_jobs = max(max_jobs - previous_jobs, 0)
+    else:
+        previous_jobs = None
+        remaining_jobs = None
+
     return max_jobs, previous_jobs, remaining_jobs
 
 
 def put_jobs(user_id: str, jobs: List[dict], fail_when_over_quota=True) -> List[dict]:
     table = DYNAMODB_RESOURCE.Table(environ['JOBS_TABLE_NAME'])
     request_time = format_time(datetime.now(timezone.utc))
 
+    priority_override = get_priority(user_id)
```

Comment:
I'd wait to declare this variable until immediately before its' used (around line 47), i.e. do all the `max_jobs` business logic, then do all the `priority` business logic